### ISSUE TITLE
USER ISSUE: Missing food section of UI

### ISSUE THREAD

**Issue by Loothor** [StrangeLoopGames/EcoIssues#22767]
\
**User Id: (Not logged user)**
**Version:** 0.9.4.3 beta release-259

**Steps to Reproduce:**
No idea.

**Expected behavior:**
It's gone

**Actual behavior:**
Should be there

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No, vanilla

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by Ry4ndrew** [comment]
Hi @Loothor
Thank you for writing this bug report. However, we are unable to reproduce this issue.
Could you make a video recording of the issue, provide game logs, and upload it here? 
This will help us further look into this issue.

**Comment by Loothor** [comment]
Hey there, fortunately it didn't happen again. I resolved it by restarting
the game.

One note, I couldn't access the skills section by hitting Z. I could eat
and get full though!

**Comment by PixlRainbow** [comment]
did you complete the tutorial?

**Comment by Loothor** [comment]
Yep, it was past the tutorial. I even tried reactivation that part of the
tutorial to get it back.
